drivers/hmis/lib/form_data/default/assessments/base_post_exit.json

```
@@ -0,0 +1,10 @@
+{
+  "item": [
+    {
+      "fragment": "#r20_post_ext_information_date"
```

```suggestion
      "fragment": "#r20_post_exit_information_date"
```

and renaming the file